yarn.lock

```diff
@@ -2314,7 +2314,15 @@ chalk@2.4.1:
     escape-string-regexp "^1.0.5"
     supports-color "^5.3.0"
 
-chalk@^1.1.1, chalk@^1.1.3:
```

Comment:
What's the reason for updating all of these?

Reply:
They might have been updated when I reinstalled create-react-app, I didn't intentionally update them.